src/clients/api/queries/getAssetsInAccount.ts

```diff
@@ -0,0 +1,18 @@
+import { Contract } from 'web3-eth-contract';
+
+export interface IGetAssetsInAccountInput {
+  comptrollerContract: Contract; // @TODO: use contract type (through Typechain?)
+  account: string | null | undefined;
+}
+
+export type IGetAssetsInAccountOutput = number;
+
+const getAssetsInAccount = async ({ comptrollerContract, account }: IGetAssetsInAccountInput) => {
+  let response = [];
+  if (account) {
```

Comment:
I think `account` should be mandatory here.
The validation should happen beforehand, so that we don't have false positive since the empty array doesn't necessarily indicate the user isn't connected.

Reply:
I agree that this would be nice but the requirements for hooks require that the number of hooks remains the same on every render which means `useUserMarketInfo` with the hooks we want must be called when account is null or undefined because we can't conditionally call them if account is defined or not.

Reply:
We can't default to an empty string here because the calls will throw unnecessary invalid address errors

Reply:
You can conditionally call a query, using the `enabled` prop: https://react-query.tanstack.com/guides/dependent-queries